packages/composables/src/useListing.ts

```diff
@@ -267,20 +267,7 @@ export function createListingComposable<ELEMENTS_TYPE>({
   const setInitialListing = async (
     initialListing: Partial<ListingResult<ELEMENTS_TYPE>>
   ) => {
-    // note: only v6.3.x compatible
-    /* istanbul ignore next */
-    if (
-      initialListing?.currentFilters?.manufacturer?.length ||
-      initialListing?.currentFilters?.properties?.length
-    ) {
-      loading.value = true;
-      const allFiltersResult = await searchMethod({
-        query: initialListing.currentFilters.search || undefined,
-      });
-      initialListing = Object.assign({}, initialListing, {
-        aggregations: allFiltersResult?.aggregations,
-      });
-    }
+    loading.value = true;
```

Comment:
Do we have to change a loader status if we only assign value in this function? This will be done in the same tick

Reply:
+1